output/output.cpp

```diff
@@ -13,6 +13,8 @@
 #include "net_output.hpp"
 #include "output.hpp"
 
+bool enable_ = true;
```

Comment:
How does this relate to the member variable of the same name?
I'm not even sure which one will be in scope, and I expect it will confused other readers as well.

Reply:
(Underscores on the end of a something normally imply a data member in a class)

Reply:
Indeed. All these funny global variables need to go, unless there is some special justification for them.